modules/library/metadata/src/test/java/org/geotools/util/GeoToolsUnitFormatRoundTripTest.java

```diff
@@ -17,15 +17,15 @@
 package org.geotools.util;
 
 import static junit.framework.TestCase.assertEquals;
-import static tec.uom.se.unit.Units.GRAM;
-import static tec.uom.se.unit.Units.JOULE;
-import static tec.uom.se.unit.Units.KELVIN;
-import static tec.uom.se.unit.Units.KILOGRAM;
-import static tec.uom.se.unit.Units.METRE;
-import static tec.uom.se.unit.Units.MOLE;
-import static tec.uom.se.unit.Units.PASCAL;
-import static tec.uom.se.unit.Units.SECOND;
-import static tec.uom.se.unit.Units.WATT;
+import static tech.units.indriya.unit.Units.GRAM;
+import static tech.units.indriya.unit.Units.JOULE;
+import static tech.units.indriya.unit.Units.KELVIN;
+import static tech.units.indriya.unit.Units.KILOGRAM;
+import static tech.units.indriya.unit.Units.METRE;
+import static tech.units.indriya.unit.Units.MOLE;
+import static tech.units.indriya.unit.Units.PASCAL;
+import static tech.units.indriya.unit.Units.SECOND;
+import static tech.units.indriya.unit.Units.WATT;
```

Comment:
I am considering something I should of probably done last time:

1. Add all the unit constants we use to geotools Units class

```java
static Unit<Distance> METRE = tech.units.indriya.unit.Units.METRE;
...
```

2. Update all of these imports with:

```java
import static org.geotools.measure.Units.*;
```

Reply:
I considered it last time but I had hoped that the implementation imports would be stable for downstream applications.

This has the advantage of any future implementation change being transparent to applications using GeoTools.

Reply:
I tried the change locally, it was not worth it since only a few test cases internal to geotools notice. And downstream applications mostly create things by text.

See https://github.com/geoserver/geoserver/search?q=tec.uom.se&unscoped_q=tec.uom.se